src/main/preferences.js

```diff
@@ -222,6 +225,24 @@ const preferencesConfig = {
 								options: [ { label: 'Lock the crosshair to the mouse cursor', value: 'followMouse' } ],
 								help: 'Keeps CrossOver centered on the mouse cursor. ',
 							},
+							{
+								label: 'Toggle Resize on ADS',
+								key: 'resizeOnADS',
+								type: 'radio',
+								options: [
+									{ label: 'Never', value: '0' },
+									{ label: 'Press right mouse-button', value: '1' },
+									{ label: 'Hold right mouse-button', value: '2' },
```

Comment:
I might also consider adding support for middle- and left-mouse buttons, but that may be beyond the scope of this PR

Reply:
Was thinking of adding support for multiple buttons and shortcuts, just in case, but held off on it